### PR TITLE
fix Wrong usage of break

### LINKED ISSUE #68332: Wrong usage of `break`
/kind bug
/sig release
`break` switch or `break` loop
`switch` do not need `break`
Sometimes we expect to break `for loop`.

Such as:
https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/cluster/images/etcd-version-monitor/etcd-version-monitor.go#L252

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/volume/flocker/flocker.go#L398

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/scheduler/core/generic_scheduler.go#L1063

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/kubectl/cmd/set/helper.go#L103

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/util/mount/mount.go#L304

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/util/mount/mount.go#L306

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/kubelet/types/pod_update.go#L95

https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/kubelet/types/pod_update.go#L97

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #68332 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/images/etcd-version-monitor/etcd-version-monitor.go [hunk 1]
```diff
@@ -243,13 +243,14 @@ func getVersion(lastSeenBinaryVersion *string) error {
...
 	lastSeenBinaryVersion := ""
+L:
 	for {
 		if err := getVersion(&lastSeenBinaryVersion); err != nil {
 			glog.Errorf("Failed to fetch etcd version: %v", err)
 		}
 		select {
 		case <-stopCh:
-			break
+			break L
```
Comment (k82cn) [inline cluster/images/etcd-version-monitor/etcd-version-monitor.go]: should be `return` ?

File: pkg/scheduler/core/generic_scheduler.go [hunk 2]
```diff
@@ -1060,7 +1061,7 @@ func nodesWherePreemptionMightHelp(nodes []*v1.Node, failedPredicatesMap FailedP
 				predicates.ErrVolumeNodeConflict,
 				predicates.ErrVolumeBindConflict:
 				unresolvableReasonExist = true
-				break
+				break L
```
Comment (k82cn) [inline pkg/scheduler/core/generic_scheduler.go]: this should be ok to keep it as no endless loop here.

**Comment by tanshanshan** [comment]
/remove-sig/scheduling

**Pull request closed without merging**